src/Cartography/Cartography.StaticMap.Shared/StaticMapControl.UWP.cs

```diff
@@ -1,10 +1,16 @@
-﻿#if NETFX_CORE
+﻿#if WINDOWS_UWP || WINDOWS
```

Comment:
Not needed since it's a uwp.cs class

Reply:
This is part of Cartography.StaticMap.WinUI which includes windows. Should I create a second class for Windows or rename the class to .Win or something of the sort?